Monika After Story/game/script-topics.rpy

```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
         m 3hua "Because I know that you're a good person despite how things are between us."
 
     elif mas_isMoniLove(higher=True):
-        m 1esc "I don't."
-        pause 3.0
-        m 1hub "Just kidding!"
-        m 1eub "You mean {i}everything{/i} to me, silly!"
-        m 1esd "But if you really want to know, [player]..."
-        # TODO: wink / tongue stick out here woul dbe perfect
-        m 1hfu "That's a secret!"
-
+        if renpy.random.randint(1,2) == 1:
+            if store.mas_anni.pastSixMonths():
```

Comment:
If you're not past 6 months here, all you get is the `return "love"`. That's an issue.
```suggestion
```